apps/modernization-ui/src/design-system/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,31 @@
+import { Pagination as PaginationUswds } from '@trussworks/react-uswds';
+import styles from './pagination.module.scss';
+import { RangeToggle } from 'components/Table/RangeToggle/RangeToggle';
+import { useEffect } from 'react';
+import { usePage } from 'page';
+
+export const Pagination = () => {
+    const { ready, request, page } = usePage();
+
+    useEffect(() => {
+        ready(page.total ?? 0, page.current);
+    }, [page]);
```

Comment:
issue:  this is why the github action is failing, it causes an infinite loop.  It's not needed, this is all handled by `usePage`...  did you run these tests locally?